### DISCUSSION TITLE
https://penguins-eggs.net/blog/un-punto-morto?fbclid=IwAR2rmdn_vYga1ZxcfUtmBx_uzgX9DLmS4hGE1GIwfBhTc3DeG3ctDsdsidY_aem_Af-LWWmKTtnWYQhEEcYjPP5-92TWCq-KKXnt6ZMzGkeAQppDaIitlM6fhV3XgZnFIt0

### DISCUSSION THREAD

**Discussion by giscus** [pieroproietti/penguins-blog#10] [General]
# https://penguins-eggs.net/blog/un-punto-morto?fbclid=IwAR2rmdn_vYga1ZxcfUtmBx_uzgX9DLmS4hGE1GIwfBhTc3DeG3ctDsdsidY_aem_Af-LWWmKTtnWYQhEEcYjPP5-92TWCq-KKXnt6ZMzGkeAQppDaIitlM6fhV3XgZnFIt0

Eggs è nato più o meno nel corso del 2016 sulle ceneri di software precedenti.

https://penguins-eggs.net/blog/un-punto-morto?fbclid=IwAR2rmdn_vYga1ZxcfUtmBx_uzgX9DLmS4hGE1GIwfBhTc3DeG3ctDsdsidY_aem_Af-LWWmKTtnWYQhEEcYjPP5-92TWCq-KKXnt6ZMzGkeAQppDaIitlM6fhV3XgZnFIt0

**Comment by fconidi** [comment]
Il tuo lavoro è stato incredibile e sicuramente dispendioso, come capita anche a me quando mi fisso a fare qualcosa. So quanto ore prende fare un lavoro del genere, portato avanti solamente dalla passione, togliendo tempo alla famiglia. Detto ciò il mio consiglio è di non tuffarti subito in una nuova avventura, poiché la stabilità di eggs viene compromessa. Negli ultimi 2 anni man mano che integravi nuove distro/derivate si sono aggiunti alcuni bugs fastidiosi, ed è impossibile per te scovarli se non ti vengono attenzionati. Il rischio quindi è molto alto, a meno che non freeze eggs per distribuzione. Il lavoro che fai da solo è enorme, quindi non puoi sapere se l’ultimo aggiornamento funziona con tutte le derivate precedenti……Fermati un po’ ;)
Grazie

**Reply by pieroproietti** [reply]
Grande @fconidi,

suggerimento prezioso, ed hai ragione, ne terrò conto!

Speriamo ne arrivino altri.
